src/pages/blog/can-llama-3-listen.mdx

```diff
@@ -0,0 +1,232 @@
+---
+title: Can llama3.1 learn to listen?
+author: ["Alan Dao", "Rex Ha", "Bach Vu", "Phong Tran"]
```

Comment:
@urmauur teh authors don't render :[

Reply:
remove 3.1

Reply:
@tikikun pls make the edit directly 😂😂😂🙏